growth/too/templates/gcn.jinja2

```diff
@@ -0,0 +1,9 @@
+{% from 'macros.jinja2' import oxford_commas %}
+
+SUBJECT: GROWTH follow-up of {{plan.event.gcn_notices[-1].stream}} trigger {{plan.dateobs}} with {{plan.telescope}}
+
+{% filter rewrap %}
+{{oxford_commas(authors)}} report on behalf of the ZTF project and the GROWTH collaboration:
+We observed the localization region of {{plan.event.gcn_notices[-1].stream}} trigger {{plan.dateobs}} UTC with {{plan.telescope}}. We obtained a series of {{oxford_commas(plan.plan_args.filt)}} band images covering {{ "%.1f"|format(plan.area) }} square degrees beginning at {{plan.validity_window_start}} ({{tdiff}} after the burst trigger time), corresponding to ~{{ "%.1f"|format(plan.probability(localization_name=plan.event.localizations[-1].localization_name)) }}% of the probability enclosed in the localization region. 
```

Comment:
You are passing the localization itself to the template, so you don't need to look it up by index. For example, replace `plan.probability(localization_name=plan.event.localizations[-1].localization_name)` with `plan.get_probability(localization)`.